### DISCUSSION TITLE
github education

### DISCUSSION THREAD

**Discussion by ao-shuo** [community/community#138447] [GitHub Education]
### Hi everyone,

As the question states, I've applied for over a three days and I haven't received my benefits yet, nor any confirmation email. I'm sure my application process is correct as shown in the attached picture. After uploading my student certification, I received "Approved" on the web page, but I still cannot use benefits. can you tell me what to do

**Comment by ferid333** [comment]
Hi!

1. **Check Your Email Spam/Junk Folder:** Sometimes confirmation emails can get filtered into spam, so it’s worth checking there.

2. **Sign Out and Back In:** Try signing out of your GitHub account and then logging back in. This can sometimes refresh your access to the benefits.

3. **Check Your Benefits Page:** Go to [GitHub Education Benefits](https://education.github.com/benefits) and see if your benefits are available there.

4. **Clear Cache or Try a Different Browser:** Sometimes browser issues can cause glitches, so try clearing your cache or accessing GitHub from a different browser.

5. **Contact GitHub Support:** If none of the above works, you can reach out to GitHub support or the GitHub Education team to ask for help.

Hopefully, one of these steps will help you access your benefits!

**Comment by davevad93** [comment]
Hi @ao-shuo ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ao-shuo was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
